bootstrap.sh

```diff
@@ -445,7 +445,8 @@ build_cmake_command(){
       set_value=OFF
       option_value="${!option}"
       if { [[ "$option_value" = "${FALSE}" ]] && { [[ "$FOUND_VALUE" == "DISABLE"* ]] || [[ "$FOUND_VALUE" == *"OFF" ]]; }; } || \
-         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" == "ENABLE"* ]]; }; then
+         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" == "ENABLE"* ]]; } ||
+         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" != "ENABLE"* ]] && [[ "$FOUND_VALUE" != "DISABLE"* ]]; }; then
```

Comment:
What is the problem that's fixed here? How did it not work?

Reply:
Enabling OpenSSL is done with the MINIFI_OPENSSL option, but only the ENABLE*/DISABLE* options were handled here, so the openssl was never set to ON regardless of the option being toggled or not.